gdpr/aggregated_config.go

```diff
@@ -50,6 +51,13 @@ func (tc *tcf2Config) IntegrationEnabled(integrationType config.IntegrationType)
 	return tc.HostConfig.Enabled
```

Comment:
You should be able to delete this function `IntegrationEnabled` and let's rename `config/config.go#IntegrationEnabled` to `config/config.go#ChannelEnabled`.